### PR TITLE
proposed webapp alpha for review

### PR DESCRIPTION
**Pull request by hubwoop**

### CODE REVIEW

**Comment by theScrabi** [comment]
I instantly get a syntax error:

```
File "/usr/local/lib/python3.5/dist-packages/matomat-0.0.0-py3.5.egg/matomat/matomat.py", line 267
    return work_view(message=f"You successfully added {amount}€ to your account!")
                                                                                ^
SyntaxError: invalid syntax
```

Is this python3 or pytho2?

File: matomat/schema.sql [hunk 1]
```diff
@@ -0,0 +1,50 @@
+-- Product table
+DROP TABLE IF EXISTS Products;
+CREATE TABLE Products
+(
+  id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
+  name TEXT NOT NULL,
+  price REAL NOT NULL,
```
Comment (theScrabi) [inline matomat/schema.sql:7]: Wouldn't it be better if we saved the price as INT?
We could save it as cent and than multiply it by 100 if we need euro. I think real is a little to much for saving money.

Comment (hubwoop) [reply matomat/schema.sql:7]: I gues you are right, shouldn't change much but isn't a lot of work either.
You need >python3.6 in order to run the alpha. You can also check out the docker branch in my repo for a dockerized version of the code. Thus, you don't need to setup your own python env.

File: run.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export FLASK_APP=matomat
```
Comment (theScrabi) [inline run.sh:1]: add a
#!/bin/sh

to the first line

**Review by theScrabi — CHANGES_REQUESTED** [review]
ok I've seen some more things that require change

File: matomat/schema.sql (hunk 1 above)
Comment (theScrabi) [reply matomat/schema.sql:7]: > check out the docker 

Why do we need docker for a matomat? That sounds a little to much for such a simple device.

> You need >python3.6

Matomat only supports python 3.5.3

**Comment by theScrabi** [comment]
When I try to run schem.sql I get these [errors](https://pastebin.com/aUY5w3LB).

**Comment by theScrabi** [comment]
Could you please come over and help setting it up, I can not get it to run, its to complicated.

File: matomat/schema.sql (hunk 1 above)
Comment (hubwoop) [reply matomat/schema.sql:7]: Its not supposed to be run on the terminal. The terminal should only serve as an endpoint. I'm still talking to AG WASA about hosting it.

File: run.sh (hunk 2 above)
Comment (hubwoop) [reply run.sh:1]: Will remove the entire file, as it's a dev relic and not needed to start the program.

**Comment by hubwoop** [comment]
Let's meet tomorrow! :)

**Comment by theScrabi** [comment]
Ow an please also put all your developed files into a sub folder like matoware_online.

**Pull request closed without merging**